presto-docs/src/main/sphinx/release/release-0.186.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.186
+=============
+
+General Changes
+---------------
+
+* Fix excessive GC overhead caused by map to map cast.
+* Fix implicit coercions for ``ROW`` types, allowing operations such as
+  comparisons between compatible types.
+* Fix issue that caused queries containing expensive functions, such as regular
```

Comment:
`caused` -> `may cause` and also `were` -> `are` ?

Reply:
I was copying a very similar release note from 0.184. But I think you're right on the tense.